matplotlib/pyplot.pyi

```diff
@@ -3,6 +3,7 @@ from matplotlib.contour import QuadContourSet
 import numpy as np
 from typing import Callable, ContextManager, Literal, Sequence, overload
 from matplotlib import rcParams as rcParams
+from matplotlib import style as style
```

Comment:
The Pylance issue seems to also say that there is something wrong with `Axes`?

Reply:
thanks. forgot about that second part

Reply:
updated